expression/builtin_string_vec.go

```diff
@@ -0,0 +1,65 @@
+package expression
+
+import (
+	"math"
+	"strings"
+
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+func (b *builtinRepeatSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	buf2, err := b.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.put(buf2)
+	if err := b.args[1].VecEvalInt(b.ctx, input, buf2); err != nil {
+		return err
+	}
+
+	result.ReserveString(n)
+	nums := buf2.Int64s()
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) || buf2.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+		num := nums[i]
+		if num < 1 {
+			result.AppendString("")
+			continue
+		}
+		if num > math.MaxInt32 {
```

Comment:
why comparing it with `math.MaxInt32`?

Reply:
I don't know and it's from the old row-based implementation.
It's better to keep it here?

Reply:
I think it is to avoid uint64 overflow at `uint64(byteLength)*uint64(num)`
We can add a comment here to make it more clear.

Reply:
Comments are added。